### PR TITLE
Plans: Implement money-back guarantee banner in HTML+SVG

### PR DESCRIPTION
**Pull request by mattgawarecki**
#### Changes proposed in this Pull Request

Resolves `1164141197617539-as-1200006697536409`.

* Replace the static images for the 14-day money-back guarantee banner with HTML and SVG content, to reduce loading times and layout shift on slower devices.

#### Known issues

* Line wrapping behavior may be slightly different than exists in production, since the current banner is a static image and this PR renders all the text HTML.
* This PR is using `breakpoint-deprecated` to handle screeen size queries, when `break-small` would be more appropriate. Unfortunately I experienced build errors when I used `break-small`, but this is something I think we can come back later to fix.

#### Testing instructions

* Open any experience (Calypso Blue, Green, Jetpack Connect) in your testing environment with this PR applied.
* Visit the appropriate Plans/Pricing page.
* Verify that the 14-day money-back guarantee displays correctly on your screen and looks similar to the one that exists in production.
* Verify that as you change the size of your browser's viewport, the banner changes size accordingly.
* Verify that at a screen width of 660 pixels, the banner transitions between a horizontal layout (for wider screens) and a vertical one (for narrower screens).

### CODE REVIEW

**Comment by robertf4** [comment]
This tests well for me. Layout shift is greatly reduced and the load time is much faster. 🚢

**Review by robertf4 — APPROVED** [review]

**Review by keoshi — APPROVED** [review]
This looks great and should be a good win for performance. Love the mobile layout too!

**Comment by keoshi** [comment]
One thing on the mobile version (though it's NOT a blocker for merging): we should probably drop the border radii since the box takes the full width of the screen, and possibly also set `border-left` and `border-right` to 0.

Before | After
--|--
image | image

**Comment by keoshi** [comment]
Thanks for making those changes so quickly, @mattgawarecki ! The code looks good to me. 🚀

Edit: just tested the changes, they look great! Launch it.

File: client/components/jetpack/intro-pricing-banner/index.tsx [hunk 1]
```diff
@@ -2,67 +2,40 @@
...
-								args: { percent: 40, days: 14 },
-							}
-						) as string
-					}
-					src={ desktop }
-				/>
-			</picture>
+			<div className="intro-pricing-banner__copy">
+				<p className="intro-pricing-banner__header">
+					{ preventWidows( translate( 'Check out our new introductory pricing' ) ) }
```
Comment (a8ci18n) [inline client/components/jetpack/intro-pricing-banner/index.tsx:22]: :information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **17** times:
`translate( 'Check out our introductory pricing' )` *ES Score: 9*

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/5560440

Thank you @mattgawarecki for including a screenshot in the description! This is really helpful for our translators.

**Comment by a8ci18n** [comment]
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/5560440).

**Pull request merged**